### PR TITLE
Add new Apple Podcasts tags

### PR DESCRIPTION
**Pull request by gcampo88**
#428

### CODE REVIEW

File: src/app/series/directives/series-podcast.component.html [hunk 1]
```diff
@@ -46,6 +46,15 @@
       </div>
     </prx-fancy-field>
 
+    <prx-fancy-field label="Podcast Type" [model]="podcast" name="serialOrder">
+      <div class="fancy-hint">Select how your episodes should be ordered: <strong>episodic</strong> (when you want your episodes to be presented and recommended last-to-first; this is the default and most common order for podcasts), or <strong>serial</strong> (when you want your episodes presented and recommended first-to-last).</div>
+      <prx-select
```
[the remark this answers is not in the document]
Comment (gcampo88) [reply src/app/series/directives/series-podcast.component.html]: ah, i thought the idea was to move towards using prx-select for any `select`s now that it's in styleguide (also was copying from the prx-select used for audio files on Episode-Basic and then just kept using in other areas). happy to change back if you say so! lmk what you think

[the remark this answers is not in the document]
Comment (sandikbarr) [reply src/app/series/directives/series-podcast.component.html]: fancy field select uses prx-select now

[the remark this answers is not in the document]
Comment (gcampo88) [reply src/app/series/directives/series-podcast.component.html]: aha! 👍

**Comment by gcampo88** [comment]
okey doke, thinks it's ready for you again @sandikbarr !

**Pull request merged**